share/doc/theory.rst

```diff
@@ -328,6 +328,56 @@ parameter a singular value decomposition of the matrix
 should dominate the smallest singular values (squared) but is itself
 small compared to the larger ones.[Rosenberger:2016]_
 
+Iterative Integral Equation methods
+-----------------------------------
+
+Iterative integral equation (IIE) methods are best compared to the IMC method.
```

Comment:
Should be

The Iterative Integral Equation

Is it a method or are there multiple methods associated with this approach?

Reply:
Multiple methods which are all pretty similar. There is the choice on the closure relation (HNC or PY) and on the Optimization method (Newton (and a modified verson of that one) or Gauss-Newton).

Reply:
Thanks for the clarification, I would still capitalize though.